src/test/java/com/jcabi/aether/ClasspathTest.java

```diff
@@ -94,6 +103,37 @@ public void buildsClasspath() throws Exception {
     }
 
     /**
+     * Classpath can build a classpath with more scopes.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void buildsClasspathWithMoreScopes() throws Exception {
+        final String vrs = "1.5.2";
+        MatcherAssert.assertThat(
+            new Classpath(
+                this.project(
+                    this.dependency(
+                        "com.jcabi", "jcabi-ssh", vrs
```

Comment:
@amihaiemil I still see test data coupling here `jcabi-ssh` string should be defined once